### ISSUE TITLE
2014 Primary Mono test failure

### ISSUE THREAD

**Issue by charles-difazio** [openelections/openelections-data-ca#108]
Missing write-in candidates and Proposition results.  Propositions are available in the [Statement of Votes Cast](http://www.monocounty.ca.gov/sites/default/files/fileattachments/elections/page/227/summary-precinct.pdf), but no luck with write-ins.

```
E           AssertionError: Mono failed
E           assert {('"Bo" Bogda...e'): 582, ...} == {('"Bo" Bogdan...e'): 582, ...}
E             Omitting 62 identical items, use -v to show
E             Left contains more items:
E             {('Jimelle L. Walls', 'Mono', '', 'Governor'): 1,
E              ('No', 'Mono', '', 'Proposition 41'): 1221,
E              ('No', 'Mono', '', 'Proposition 42'): 1378,
E              ('Patrick D. Hogan', 'Mono', 5.0, 'State Assembly'): 17,
E              ('Yes', 'Mono', '', 'Proposition 41'): 1709,
E              ('Yes', 'Mono', '', 'Proposition 42'): 1455}
E             Use -v to get the full diff
```

**Comment by carbonphyber** [comment]
Fixed in #152 

Note: this PR doesn't fix the lack of precinct-level data for write-ins (see the last 2 lines) but creates new "Unknown" precinct to be able to fix the tests. Please provide feedback on that.

**Comment by warwickmm** [comment]
I'm not sure what the convention is for a "missing" or "unknown" value.  It appears that we often leave the entry blank in these cases.  For example,
```
Mono,,Governor,,,Jimelle L. Walls,1
Mono,,State Assembly,5,LIB,Patrick D. Hogan,17
```
instead of
```
Mono,Unknown,Governor,,,Jimelle L. Walls,1
Mono,Unknown,State Assembly,5,LIB,Patrick D. Hogan,17
```
Until we establish a convention, I think the current solution is fine.
